cli/config/configuration-qemu-virtiofs.toml.in

```diff
@@ -16,6 +16,19 @@ kernel = "@KERNELVIRTIOFSPATH@"
 image = "@IMAGEPATH@"
 machine_type = "@MACHINETYPE@"
 
+# List of valid annotation names for the hypervisor
+# Each member of the list is a regular expression, which is the base name
+# of the annotation, e.g. "path" for io.katacontainers.config.hypervisor.path"
+# The default if not set is empty (all annotations rejected.)
+# Your distribution recommends: @DEFENABLEANNOTATIONS@
+enable_annotations = @DEFENABLEANNOTATIONS@
+
+# List of valid annotations values for the hypervisor
```

Comment:
sorry this is confusing me:  _List of valid **annotations** values for the hypervisor_  - or -   _List of **paths** for the hypervisor_ ? :thinking: 
an annotation look like this: "io.kata.feature", a path looks like this: "/usr/bin/qemu"

Reply:
An annotation looks like `io.kata.feature-name: feature-value`. These are called "name-value" pairs, so I refer to the value for the annotation here.

Does that make sense?

(The fact that the value is a path here is relevant, so I may change the wording to reflect that).

Reply:
> (The fact that the value is a path here is relevant, so I may change the wording to reflect that).

However, re-reading, I'm not sure how to improve the wording. Any suggestion?

Reply:
might be  _List of paths for the hypervisor_   ..?

Reply:
Updated the comments in a separate commit in order to be able to forward-port to 2.0 where this is already merged.

Reply:
Done